strr-web/components/bcros/form-section/property/Details.vue

```diff
@@ -91,6 +87,65 @@
           />
         </UFormGroup>
       </div>
+      <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
+        <UFormGroup name="rentalUnitSpaceType" class="d:pr-[16px] flex-grow" :error="rentalUnitSpaceTypeError">
+          <USelect
+            v-model="rentalUnitSpaceType"
+            :placeholder="t('createAccount.propertyForm.rentalUnitSpaceType')"
+            :options="rentalUnitSpaceTypeOptions"
+            style="color: #1a202c; /* text-gray-900 */ dark:text-white; /* Override with dark mode text color */"
+            @blur="emit('validateRentalUnitSpaceType')"
+            @change="emit('validateRentalUnitSpaceType')"
+          />
+        </UFormGroup>
+      </div>
+      <div class="flex flex-row justify-between w-full mb-[40px] mobile:mb-[16px]">
+        <UFormGroup name="isUnitOnPrincipalResidenceProperty" class="d:pr-[16px] flex-grow">
+          <USelect
+            v-model="isUnitOnPrincipalResidenceProperty"
+            :placeholder="t('createAccount.propertyForm.isUnitOnPrincipalResidenceProperty')"
+            :options="principalResidenceOptions"
+            class="w-full"
+            style="color: #1a202c; /* text-gray-900 */ dark:text-white; /* Override with dark mode text color */"
+            @blur="emit('validateIsUnitOnPrincipalResidenceProperty')"
+            @change="(value) => { isUnitOnPrincipalResidenceProperty = (value === 'true') }"
```

Comment:
This is not needed because `v-model` is already handling the value updates.

Reply:
maybe i missed something but v-model doesn't handle when a user first selects yes and then selects no. So in that case the question 3 remains.